src/lib/assets/assets.test.ts

```diff
@@ -261,4 +263,39 @@ describe("Assets", () => {
     await assets.generateHelmChart(webhookGeneratorFunction, "/tmp");
     expect(consoleErrorSpy).not.toHaveBeenCalled();
   });
+
+  it("should call generateHelmChart and get and not get an error when asset class instance is correct", async () => {
```

Comment:
```suggestion
 it("should call generateHelmChart without an error when asset class instance is correct", async () => {
```

Rewording test name for clarity.